### PR TITLE
ts/indicators-interfaces

### PR DESCRIPTION
**Pull request by sebastianbochan**
Fixed missing properties in interfaces.

### CODE REVIEW

File: ts/Stock/Indicators/PC/PCIndicator.ts [hunk 1]
```diff
@@ -179,10 +179,13 @@ class PCIndicator extends SMAIndicator implements Highcharts.MultipleLinesIndica
...
 interface PCIndicator {
-    getTranslatedLinesNames: Highcharts.MultipleLinesMixin[
+    getTranslatedLinesNames: typeof MultipleLinesMixin[
         'getTranslatedLinesNames'
     ];
-    linesApiNames: Highcharts.MultipleLinesMixin['linesApiNames'];
+    drawGraph: typeof MultipleLinesMixin.drawGraph;
+    toYData: typeof MultipleLinesMixin.toYData;
+    translate: typeof MultipleLinesMixin.translate;
+    linesApiNames: typeof MultipleLinesMixin['linesApiNames'];
```
Comment (bre1470) [inline ts/Stock/Indicators/PC/PCIndicator.ts:188]: Only use brackets when not using typeof.

**Review by bre1470 — CHANGES_REQUESTED** [review]
Nice! One thing...

File: ts/Stock/Indicators/PC/PCIndicator.ts (hunk 1 above)
Comment (sebastianbochan) [reply ts/Stock/Indicators/PC/PCIndicator.ts]: Fixed

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Pull request merged**